control/control.SLES.xml

```diff
@@ -303,21 +302,24 @@ defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</l
           <label>Default System</label>
         </normal_role>
         <normal_role_description>
-          <label></label>
+          <label>• GNOME environment, with BTRFS root (/) partition
+• Separate /home partition (xfs) for disks larger than 20GB</label>
```

Comment:
It seems the preferred spellings are Btrfs and XFS:

- https://en.wikipedia.org/wiki/Btrfs
- https://en.wikipedia.org/wiki/XFS